plugins/WindowExplorer/wndprp.c

```diff
@@ -1030,7 +1033,7 @@ BOOL CALLBACK EnumPropsExCallback(
     {
         PPH_STRING propName;
 
-        propName = PhFormatString(L"#%lu", (ULONG_PTR)lpszString);
+        propName = PhFormatString(L"#%lu", (USHORT)lpszString);
```

Comment:
The format specifier has %lu for unsigned long when you've cased to USHORT

Reply:
True, but we just checked that it fit in a short. The format specifier should probably be changed (?)